cmd/brew-services.rb

```diff
@@ -270,30 +255,21 @@ def cleanup
     end
 
     # Stop if loaded, then start again
-    def restart
-      services = target_services
-      @act_on_all_services = false
-
-      services.each do |service|
-        @target_services = [service]
-        @single_service = service
-
-        stop if service.loaded?
-        start
+    def restart(target)
+      Array(target).each do |service|
```

Comment:
Not disagreeing but what's the advantage over `target.to_a`? I've always wondered.